composer.json

```diff
@@ -18,7 +18,7 @@
         "ext-json": "*",
         "php": ">=7.3",
         "psr/log": ">=1.0",
-        "guzzlehttp/guzzle": ">=6.5",
+        "guzzlehttp/guzzle": ">=6.5 <=7.5.0",
```

Comment:
I think this can be `<8` right? Because according to guzzle devs this will be deprecated in version 8